corehq/apps/app_manager/views/utils.py

```diff
@@ -663,14 +663,16 @@ def append_endpoint(endpoint_id):
         append_endpoint(endpoint)
 
     for module in app.modules:
-        if module.unique_id != module_id:
+        if module.unique_id != module_or_form_id:
```

Comment:
If I'm reading this correctly, it means when editing a module, we'll skip checking the forms inside the module as well as the module itself.  I think that's incorrect behavior (though it predates this PR)

Reply:
great point, thank you - 2afa2d42705

Reply:
It looks like this was a good fix and the remaining test failures are because the same session endpoint id is being used multiple times in these tests: https://github.com/dimagi/commcare-hq/blob/46d12c22b9257408a7496b4e60ea68bb20f79559/corehq/apps/app_manager/tests/views/test_utils.py#L126-L129